inject/src/main/java/io/micronaut/context/DefaultBeanContext.java

```diff
@@ -2448,7 +2448,7 @@ protected void processParallelBeans(List<BeanDefinitionProducer> parallelBeans)
                             loadEagerBeans(producer, parallelDefinitions);
                         } catch (Throwable e) {
                             BeanDefinitionReference<Object> beanDefinitionReference = producer.getReference();
-                            LOG.error("Parallel Bean definition [" + beanDefinitionReference.getName() + MSG_COULD_NOT_BE_LOADED + e.getMessage(), e);
+                            LOG.error("Parallel Bean definition [{}{}{}", beanDefinitionReference.getName(), MSG_COULD_NOT_BE_LOADED, e.getMessage(), e);
```

Comment:
Should the log message here contain a closing bracket?
```suggestion
                            LOG.error("Parallel Bean definition [{}{}{}]", beanDefinitionReference.getName(), MSG_COULD_NOT_BE_LOADED, e.getMessage(), e);
```